modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentConfiguration.java

```diff
@@ -309,7 +319,7 @@ public AgentConfiguration nodeKeyStore(String nodeKeyStore) {
      * @return Node key store password.
      */
     public String nodeKeyStorePassword() {
-        return nodeKeyStorePass;
+        return !F.isEmpty(nodeKeyStorePass) ? nodeKeyStorePass : getPasswordFromKeyStore("node-key-store-password");
```

Comment:
Lets invert condition to not use negate operator.

Reply:
Done.